doc/docs/master/deploy-manage/manage/upgrades.md

```diff
@@ -126,54 +116,62 @@ shows all pods as `READY`:
 
-- **A - (RECOMMENDED in production) Create your secret(s) ahead of your cluster creation** 
+- **A - (RECOMMENDED in production) Create your secret(s) ahead of time** 
       
       **Create your secret(s) ahead of your cluster creation** and provide each secret name in your values.yaml at the time of the first installation. 
 
-      Find the list of Secret Keys for each setting and their corresponding secret name field in your values.yaml (Column A) in the table below.
+      For each set, find the secret name field that references your secret in your values.yaml (Column A) and its corresponding Secret Key (First column) in the table below.
+
+!!! Note
+      A reminder that if you are using a pre-built PostgreSQL, you will not be able to pre-create a secret holding your password. It will be generated for you during the first installation and must be retrieved and set back at each upgrade.
+
 
 - **B - Provide credentials directly in your values.yaml**
 
-      You do not want to create your secrets right away, but provide specific secret credentials directly in your values.yaml or by setting them  with a `--set` argument during the upgrade.
+    If you do not want to create your secrets right away, you can provide specific  credentials directly in your values.yaml (Column B lists the names of the fields in which you can set those values directly), or put them with a `--set` argument during the installation/upgrade.
 
-      In this case, (see column B), we will read then insert those values in a default secret `pachyderm-bootstrap-config` for you. The associated keys in `pachyderm-bootstrap-config` are listed in column C of the following table.
 
+|Secret KEY name| <div style="width:290px"> Description </div>| A - Create your secrets ahead <br> of your cluster creation| B - Pass credentials in values.yaml| <div style="width:250px"> C - Corresponding (Pachyderm Secret, Key) in which the values provided in A or B will be injected. See the following `Pachyderm Internal Secrets` section for details.</div>| 
+|------------|------------|-----|--------|---------|
+|root-token| Root clusterAdmin| pachd.rootTokenSecretName |pachd.rootToken|(pachyderm-auth, rootToken)|
+|root-token|Users give this token in their values.yaml when they install a cluster that is under an [Enterprise Server's](../../enterprise/auth/enterprise-server/setup) umbrella.|pachd.enterpriseRootTokenSecretName|pachd.enterpriseRootToken|????|
+|postgresql-password|Password to your database|global.postgresql.postgresqlExistingSecretName <br> global.postgresql.postgresqlExistingSecretKey |global.postgresql.postgresqlPassword|(postgres, postgresql-password) , (postgres, postgresql-postgres-password)|
+|OAUTH_CLIENT_SECRET|Oauth client secret for Console <br> Required if you set Console|console.config.oauthClientSecretSecretName |console.config.oauthClientSecret|(pachyderm-console-secret, OAUTH_CLIENT_SECRET)|
```

Comment:
We should mention that this can be any random value.  I have been using:

```
$ openssl rand -base64 42
tJkHm0+8niOtP1F8lAPryO9dGwMV7SL/u/uCZQi24kFuRj+7VYvtj01q
```
I pick -base64 to get more entropy per byte-of-length than -hex (`openssl rand` doesn't have an alphanumeric option like helm does), but I'm not using base64 as base64; i.e. I wouldn't run "base64 -d" on this and make that the real value (which is what would happen if you were editing a secret and put it in `data` instead of `stringData`).  This point might confuse users that know k8s secrets use base64, that's totally unrelated here, I'm just trying to get the best thing out of openssl I can.  Might be too confusing.